### ISSUE TITLE
Question: Is there a roadmap

### ISSUE THREAD

**Issue by rabol** [BRACKETS-by-TRIAD/craftable#183]
Hi

Is there a roadmap of what is coming ?
e.g. is there a admin-role package in the works ?

**Comment by palypster** [comment]
Good point, we have internal roadmap, maybe we can share it with the world. We will discuss it internally first.

Role management is on our list, but I cannot promise it will be done soon (there are higher priorities).
